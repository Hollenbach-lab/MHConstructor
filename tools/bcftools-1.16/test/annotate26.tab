1	101	111
1	102	222
1	103	333
