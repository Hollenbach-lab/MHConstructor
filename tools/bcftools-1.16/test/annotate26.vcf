##fileformat=VCFv4.2
##contig=<ID=1,length=248956422>
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO
1	101	.	G	.	.	.	.
1	102	.	G	.	.	.	.
1	103	.	G	.	.	.	.
