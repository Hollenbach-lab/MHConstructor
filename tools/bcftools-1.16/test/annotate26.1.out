##fileformat=VCFv4.2
##FILTER=<ID=PASS,Description="All filters passed">
##contig=<ID=1,length=248956422>
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO
1	111	.	G	.	.	.	.
1	222	.	G	.	.	.	.
1	333	.	G	.	.	.	.
